.travis.yml

```diff
@@ -28,6 +28,7 @@ cache:
 before_install:
```

Comment:
Does Gradle need larger heap because of the multi-dexing that was turned on recently?